delivery_recipe/inputs_to_delivery.md

```diff
@@ -1,47 +1,32 @@
-# Inputs to Delivery
+# Inputs to Build
 
-To deliver change we have a clear process with Scrum. To support that process and the decisions made within it, we require certain inputs. These inputs are initially identified as part of the [discovery](/delivery_recipe/Discovery/README.md) and will be handed over to the delivery team to help them understand what they're building, why and who for.
+To deliver change we have a clear process with Scrum. To support that process and the decisions made within it, we require certain inputs. These inputs are initially identified as part of the [discovery](/delivery_recipe/Discovery/README.md), are then tested and refined in [prototyping](//delivery_recipe/Prototyping/README.md), and will now be fed into the build phase in which the service will begin to be built to production quality.
 
-## Input: Business Goals
+## Business Goals
 
-While we focus the requirements on the user the business will also have specific goals that it is trying to achieve through this period of change. Business goals could include increasing revenue, attracting more customers or opening up their online business to a wider audience. 
+While we focus the requirements on the user the business will also have specific goals that it is trying to achieve through this period of change. Business goals could include increasing revenue, attracting more customers or opening up their online business to a wider audience.
 
 These goals will help us identify the high-level requirements that will need to be delivered in order to achieve the business goals.
 
-## Input: User Needs
```

Comment:
We're passing business goals into build but not user needs? It feels like we'd pass in both as high level context or neither?